Service/ZGWService.php

```diff
@@ -226,6 +226,36 @@ public function drcLockHandler(array $data, array $configuration): array
 
         $objectEntity = $this->entityManager->getRepository('App:ObjectEntity')->find($path['id']);
         if ($objectEntity instanceof ObjectEntity) {
```

Comment:
You can remove 1 nesting level by changing this if statement to 
```
if ($objectEntity instanceof ObjectEntity === false) {
  return $this-data;
}
```

It will look better as well